### PR TITLE
Reduce deprecation warnings from www

### PR DESCRIPTION
**Pull request by uranusjr**
Some things I found after disabling `ignore::DeprecationWarning` in #20322. Not all.

### CODE REVIEW

File: airflow/www/templates/airflow/dag.html [hunk 1]
```diff
@@ -36,7 +36,7 @@
   <meta name="extra_links_url" content="{{ url_for('Airflow.extra_links') }}">
   <meta name="paused_url" content="{{ url_for('Airflow.paused') }}">
   <meta name="tree_data" content="{{ url_for('Airflow.tree_data') }}">
-  <meta name="is_paused" content="{{ dag.is_paused }}">
+  <meta name="is_paused" content="{{ dag.get_is_paused() }}">
```
Comment (bbovenzi) [inline airflow/www/templates/airflow/dag.html]: I believe there are a few more places in `/www` that we use `is_paused`

**Comment by uranusjr** [comment]
```
'airflow.models.dag.DagModel object' has no attribute 'get_is_paused'
```

Uh.

File: tests/www/test_app.py [hunk 2]
```diff
@@ -240,7 +240,8 @@ def test_should_set_permanent_session_timeout(self):
     @conf_vars({('webserver', 'cookie_samesite'): ''})
     @dont_initialize_flask_app_submodules
     def test_correct_default_is_set_for_cookie_samesite(self):
-        app = application.cached_app(testing=True)
+        with pytest.deprecated_call():
+            app = application.cached_app(testing=True)
```
Comment (ashb) [inline tests/www/test_app.py:245]: What is giving deprecation warnings here?

**Comment by ashb** [comment]
> ```
> 'airflow.models.dag.DagModel object' has no attribute 'get_is_paused'
> ```
> 
> Uh.

DAG vs DagModel maybe?

File: tests/www/test_app.py (hunk 2 above)
Comment (uranusjr) [reply tests/www/test_app.py:245]: The default value of `cookie_samesite` was an empty string, but was changed to `Lax` in 2.0. Setting the config to an empty string now emits a deprecation warning, and the value is automatically corrected to `Lax`. I’ll add a note as a code comment.

File: airflow/www/templates/airflow/dag.html (hunk 1 above)
Comment (uranusjr) [reply airflow/www/templates/airflow/dag.html]: All replaced now! (Actually only a handful across two files.)

**Comment by uranusjr** [comment]
Alright, working now. (The provider package build is failing due to #20420)

**Review by kaxil — APPROVED** [review]

**Pull request merged**